api/Navigator.json

```diff
@@ -55,53 +55,112 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/authentication",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "67",
+              "flags": [
+                {
+                  "name": "WebAuth",
+                  "type": "preference"
+                }
+              ]
             },
             "chrome_android": {
-              "version_added": "40"
+              "version_added": "67",
+              "flags": [
+                {
+                  "name": "WebAuth",
+                  "type": "preference"
+                }
+              ]
```

Comment:
It will only be on by default in desktop Chrome. I'm trying to find out what version the flag landed in.

Reply:
That was merged as part of #1742.